testsuite/tests/kuadrant/defaults/test_basic.py

```diff
@@ -0,0 +1,45 @@
+"""Test basic enforcement of the rules inside the 'defaults' block of the AuthPolicy and RateLimitPolicy"""
+
+import pytest
+
+from testsuite.httpx.auth import HttpxOidcClientAuth
+from testsuite.policy.rate_limit_policy import Limit
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
+
+LIMIT = Limit(3, 5)
```

Comment:
Why has this been defined separately as a global variable?

Reply:
For consistency across usages. Fixture seems too expansive for this. Tell me if you know of any disadvantages

Reply:
Makes sense, LGTM